recipes/poco/all/conanfile.py

```diff
@@ -106,10 +106,6 @@ def config_options(self):
     def configure(self):
         if self.options.enable_apacheconnector:
             raise ConanInvalidConfiguration("Apache connector not supported: https://github.com/pocoproject/poco/issues/1764")
-        if self.options.enable_data_mysql:
```

Comment:
Why not removing those options for Poco < 1.10? They affect the package for nothing

Reply:
the options are already there, before PR. I am only adding processing of them.

Reply:
see https://github.com/conan-io/conan-center-index/blob/80c73584466586afbb85458a5770cf5d6be3e5ca/recipes/poco/all/conanfile.py#L103